index.bs

```diff
@@ -695,7 +695,39 @@ For instance, an {{MLOperand}} may represent a constant feeding to an operation
 interface MLOperand {};
 </script>
 
-See also [[#security-new-ops]]
+{{MLOperand}} has the following internal slots:
+<dl dfn-type=attribute dfn-for="MLOperand">
+    : <dfn>\[[kind]]</dfn> of type `enum { "input", "constant", "output"}`
+    ::
+        The {{MLOperand}}'s kind
+    : <dfn>\[[type]]</dfn> of type {{MLOperandType}}
+    ::
+        The {{MLOperand}}'s type
+    : <dfn>\[[dimensions]]</dfn> of type `sequence<unsigned long>`
+    ::
+        The {{MLOperand}}'s dimensions
+    : <dfn>\[[name]]</dfn> of type [=string=]
+    ::
+        The {{MLOperand}}'s name (only for input operands)
+    : <dfn>\[[tensorData]]</dfn> of type {{ArrayBufferView}}
+    ::
+        The {{MLOperand}}'s data buffer (only for constant operands)
+    : <dfn>\[[activation]]</dfn> of type {{MLActivation}}
```

Comment:
`MLOperand` may not need to associate with an `MLActivation`. It may need a slot for the operator (e.g., platform object represents the node within the platform graph) that produces this `MLOperand`.

Reply:
Yes, I agree this was meant as an internal slot for the operator. But currently MLOperator is MLActivation. I think we might need both. I expressed my doubts in #335